htdocs/website/websiteaccount_card.php

```diff
@@ -32,9 +32,9 @@
 $langs->loadLangs(array("website", "other"));
 
 // Get parameters
-$id = GETPOST('id', 'int');
+$id         = GETPOST('id', 'int');
```

Comment:
Note that adding a space is useless because tool to sanitize and standardize code and a lot of IDE remove them to keep only 1 space. So this change will probably be reverted by another PR done by someone else.

Reply:
unterstood
but that is a pity, because it allows better & faster recognition of patterns and errors